### PR TITLE
feature authentication

### PR DESCRIPTION
**Pull request by noalexan**
- creating backend
- user authentication
- adding `generate-env.sh`

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -9,19 +9,22 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "jsonwebtoken": "^9.0.2",
```
Comment (Nimpoo) [inline frontend/package.json:12]: EQJy2ROVUAEIYg8

**Review by Nimpoo — COMMENTED** [review]
chokbar de OUF

**Review by jumunozz — APPROVED** [review]
Noice

**Review by Nimpoo — APPROVED** [review]

**Review by Vodki — APPROVED** [review]
POLOLO

**Pull request merged**